### PR TITLE
VOMS 3.0.6 

### PR DESCRIPTION
**Pull request by maarten-k**
Update of VOMS Clients to version 3.0.6 which is build with Maven

### CODE REVIEW

File: Library/Formula/voms.rb [hunk 1]
```diff
@@ -12,14 +11,19 @@ class Voms < Formula
...
   depends_on "openssl"
 
   def install
-    system "./autogen.sh"
-    system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}"
-    system "make", "install"
+    system "mvn", "package"
+    system "tar", "-xf", "target/voms-clients.tar.gz"
+    share.install "voms-clients/share/java"
+    man.mkpath
```
Comment (bfontaine) [inline Library/Formula/voms.rb]: You can remove this line. `man5.install` below will create the required directories.

File: Library/Formula/voms.rb [hunk 2]
```diff
@@ -12,14 +11,19 @@ class Voms < Formula
...
+    man5.install Dir["voms-clients/share/man/man5/*.5"]
+    man1.install Dir["voms-clients/share/man/man1/*.1"]
+    bin.install Dir["voms-clients/bin/*"]
+  end
+  test do
```
Comment (bfontaine) [inline Library/Formula/voms.rb]: Please leave an empty line between the `install` method and the `test do` block. Thanks!

**Comment by apjanke** [comment]
Merged (and squashed to single commit). Thank you for your contribution to Homebrew!

**Comment by maarten-k** [comment]
merged in 9016c92da13d722ff0526f8765b4ca56f10a3169

**Pull request closed without merging**